### PR TITLE
Updating carousel

### PR DESCRIPTION
**Pull request by PaulKillian**

### CODE REVIEW

**Comment by PaulKillian** [comment]
I messed up this commit. I was having git trouble and I couldn't switch from the carousel branch to the photos branch. Sorry for the mixed code!!!

**Comment by PaulKillian** [comment]
screencapture-192-168-0-102-3000-2020-12-28-16_49_45
screencapture-192-168-0-102-3000-2020-12-28-16_48_53
screencapture-192-168-0-102-3000-2020-12-28-16_49_20

File: src/components/Carousel/index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import './carousel.css'
+
+class Carousel extends React.Component {
```
Comment (MichaelPooleLF) [inline src/components/Carousel/index.js:4]: Might rework this into a react hook later.

**Review by MichaelPooleLF — APPROVED** [review]
Looks good, we might later rework the class to utilize react hooks, but as long as it works, it works!

**Pull request merged**